votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -35,10 +35,23 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  James self-nominated to the Moderation Team. A Node.js Collaborator has
+  communicated privately they are objecting to that nomination, they are not
+  sharing publicly the reasoning.
+  Per our Moderation Policy, the nomination cannot proceed without a TSC vote.
+  
+  Refs: https://github.com/nodejs/TSC/issues/1773#issuecomment-3131028605
 
 # 6. Optionally, insert a brief introduction for the vote PR, in the markdown format.
 prBody: |
+  The ballots from this vote should not be communicated publicly, as suggested in
+  https://github.com/nodejs/TSC/issues/1773#issuecomment-3137209359.
+  After the vote has been opened for seven days and/or all TSC voting members have
+  sent an encrypted ballot, the secret parts should be communicated to the
+  Moderation Team which would be charged to communnicate the result, while keeping
+  the ballots secret.
```

Comment:
@mcollina the goal is for the vote to be anonymous

Reply:
The moderation team could see the individual votes then, no?

Reply:
That's correct, whoever we send the secret parts to would be able to read the ballots. I'm suggesting the moderation team as I believe they would be seen as a trusted entity – it doesn't have to be the whole Moderation Team, but IMO it's better if it's not a single individual.

Reply:
Agree with @aduh95 on this. I would trust anyone currently on the moderation team to view the results and trust that they'd keep it a secret. The one thing that I would want to make sure of is that if I do join the moderation team, those results are not kept in a location where I'd be able to see them after the fact.